### PR TITLE
Enable PDBs for the runtime dlls.

### PR DESCRIPTION
**Pull request by hjyamauchi**

### CODE REVIEW

**Comment by hjyamauchi** [comment]
@tristanlabelle @compnerd

File: build.ps1 [hunk 1]
```diff
@@ -774,6 +774,7 @@ function Build-Runtime($Arch)
         SWIFT_PATH_TO_LIBDISPATCH_SOURCE = "$SourceCache\swift-corelibs-libdispatch";
         SWIFT_PATH_TO_STRING_PROCESSING_SOURCE = "$SourceCache\swift-experimental-string-processing";
         SWIFT_PATH_TO_SWIFT_SYNTAX_SOURCE = "$SourceCache\swift-syntax";
+        CMAKE_SHARED_LINKER_FLAGS = "/DEBUG /INCREMENTAL:NO /OPT:REF";
```
Comment (compnerd) [inline build.ps1:777]: I think we should also enable ICF?

**Review by tristanlabelle — APPROVED** [review]

File: build.ps1 (hunk 1 above)
Comment (hjyamauchi) [reply build.ps1:777]: I think the ICF flag caused an error somewhere during the build. I'll get the error message.

Comment (hjyamauchi) [reply build.ps1]: Nope. This doesn't happen anymore. Maybe an incremental build glitch.

Enabled ICF.

**Review by compnerd — APPROVED** [review]

**Pull request merged**